irs_lqr/irs_lqr.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+import time
+
+from irs_lqr.tv_lqr import TvLqr, get_solver
+
+class IrsLqr():
```

Comment:
nit: class definition doesn't need the pair of brackets `()` after the class name.

Reply:
Done.